app/controllers/items_controller.rb

```diff
@@ -17,13 +18,18 @@ def show
   end
 
   def new
-    @item = Item.new(status: 'Available')
+    @item_type = ItemType.find ( params[:item_type_id])
```

Comment:
I don't think there's anything really "wrong" with this method but it doesn't feel right to me. 

It's weird that you get the `@item_type` as a stand-alone thing even though it's theoretically part of the `@item` that we're working with.

Similarly it feels odd that it's being used as it is in the view layer. I think I would rather see:

```

item_type = ItemType.find( params[:item_type_id])
@item = Item.new(status: 'Available',
                               item_type: item_type)
```